napi.h

```diff
@@ -491,6 +491,26 @@ namespace Napi {
       double numberValue           ///< Property value
     );
 
+    /// Delete property.
+    bool Delete(
+      napi_value key ///< Property key primitive
+    );
+
+    /// Delete property.
+    bool Delete(
+      Value key ///< Property key
+    );
```

Comment:
I don’t think we need this key since `Value` has an `operator napi_value` member, right?

Reply:
Indeed I guess so; I just copied the definitions from `Object::Has`. Should I remove all of them?

Reply:
I think I had to add the overloads that take `Value` in order to avoid compiler errors in some situations about ambiguous calls to overloaded methods.

Reply:
@jasongin I think it should be safe to remove those – if you’re talking about #86, that’s happening because there were two multiple arguments that were used for overloading …

@rolftimmermans I’m not sure, but it certainly doesn’t have to happen in this PR :)

Reply:
Oh, wait, maybe that is about `String`s that also have an `operator std::string`? That would make sense but we should have tests for that :)